test/fixtures/oneof/required.apib

```diff
@@ -0,0 +1,13 @@
+# GET /ab
+- response 200 (application/json)
+    - Attributes (Test)
+
+# Data Structures
+
+## Test (object)
+- One Of
+    - Properties
+        - foo: A (string, required)
+
+    - Properties
```

Comment:
Try a test case with removing only the above line. `Properties` is not always needed.